src/main/java/net/minecraftforge/common/util/LazyOptional.java

```diff
@@ -29,6 +29,10 @@
 import javax.annotation.Nullable;
 import javax.annotation.ParametersAreNonnullByDefault;
 
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
```

Comment:
I think this is no longer needed

Reply:
Whoops, I forgot to remove the logger. Done.